src/conditionals/deactivating-yoast-seo-conditional.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Yoast\WP\SEO\Conditionals;
+
+/**
+ * Conditional that is only met when on deactivating Yoast SEO.
+ */
+class Deactivating_Yoast_Seo_Conditional implements Conditional {
+
+	/**
+	 * Returns whether or not this conditional is met.
+	 *
+	 * @return bool Whether or not the conditional is met.
+	 */
+	public function is_met() {
+		// phpcs:ignore WordPress.Security.NonceVerification -- We can't verify nonce since this might run from any user.
+		if ( isset( $_GET['action'] ) && \sanitize_text_field( \wp_unslash( $_GET['action'] ) ) === 'deactivate' && isset( $_GET['plugin'] ) && \sanitize_text_field( \wp_unslash( $_GET['plugin'] === 'wordpress-seo/wp-seo.php' ) ) ) {
```

Comment:
Does this mean that we'd be fixing it only if the directory's name is unaltered?

Reply:
I think so, but I also think that if the directory's name is unaltered there are going to be bigger problems than this